lib/task_helpers/imports.rb

```diff
@@ -4,6 +4,20 @@ def self.parse_options
       require 'trollop'
       options = Trollop.options(EvmRakeHelper.extract_command_options) do
         opt :source, 'Directory or file to import from', :type => :string, :required => true
+        opt :overwrite, 'Overwrite existing objects', :type => :boolean, :short => 'o', :default => false
```

Comment:
Could `parse_options` yield `opt`?  Then the caller could pass a block that defines additional parameters?

Then the caller would just need a block to define `opt :overwrite...` instead of duplicating this whole method.  Seems different imports types requiring different parameters should not be a unique concept.